.github/workflows/ci.yml

```diff
@@ -14,8 +14,8 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.23.8'
-  GOLANGCI_VERSION: 'v1.62.0'
+  GO_VERSION: '1.24.1'
```

Comment:
go1.24.3 was a security release so if updating to 1.24, it should be 1.24.3 (or [.4 in a couple of days](https://groups.google.com/g/golang-announce/c/4wOcMUma2Vc))